lib/game.rb

```diff
@@ -0,0 +1,72 @@
+# frozen_string_literal: true
+
+require_relative 'player.rb'
+
+# class tic-tac-toe game
+class Game
+  attr_accessor :board, :end_game
+  def initialize
+    @board = [' ', ' ', ' ', ' ', ' ', ' ', ' ', ' ', ' ']
+    @player1 = Player.new('Player 1', 'X')
+    @player2 = Player.new('Player 2', 'O')
+    @current_player = @player1
+    @end_game = false
+  end
+
+  def draw_board
+    puts "\n"
+    puts " #{@board[0]} | #{@board[1]} | #{@board[2]}"
+    puts '-----------'
+    puts " #{@board[3]} | #{@board[4]} | #{@board[5]}"
+    puts '-----------'
+    puts " #{@board[6]} | #{@board[7]} | #{@board[8]}"
+  end
+
+  def draw_legend
+    puts 'Legend'
+    puts '1 | 2 |3 '
+    puts '4 | 5 |6 '
+    puts '7 | 8 |9 '
+    puts "\n"
+  end
+
+  def which_player?
+    @current_player = @current_player == @player1 ? @player2 : @player1
+  end
+
+  def player_input
+    puts "Your turn, #{@current_player.name}. Pick a cell by
+    choosing a number between 1-9."
+    input = gets.chomp.to_i
```

Comment:
`main.rb` file should be responsible for user interaction. This is the only one place where puts and get.chomp can be used._